crates/matrix-sdk-crypto/src/machine/tests/decryption_verification_state.rs

```diff
@@ -393,15 +397,13 @@ async fn test_decryption_trust_requirement() {
     let room_id = room_id!("!test:example.org");
     let (event, session_id) = encrypt_message(&alice, room_id, &bob, "Secret message").await;
 
-    // Set the sender data to various values, and test that we can or can't
-    // decrypt, depending on what the trust requirement is.
-
+    // Set the SenderData on the megolm session used to encrypt `event` to
+    // `DeviceInfo` (ie,  we have the device keys but no cross-signing
+    // information). Events sent on such a session should be decryptable only
+    // when the trust requirement allows untrusted or legacy sessions.
     let mut session =
         bob.store().get_inbound_group_session(room_id, &session_id).await.unwrap().unwrap();
 
```

Comment:
nit: this empty line feels a bit unhelpful, given the comment at lines 400-403 applies as much to the bit below the blank line as it does to lines 404-405.